code/modules/mob/living/carbon/human/species/station/alraune.dm

```diff
@@ -92,8 +94,7 @@
 		)
 
 /datum/species/alraune/can_breathe_water()
-	return TRUE //eh, why not? Aquatic plants are a thing.
-
+	return waterbreathing //eh, why not? Aquatic plants are a thing.
```

Comment:
Above, make the highest can_breathe_water() just return that var, don't define on any particular species.

Reply:
So remove all species specific can_breath_water?

Reply:
Pretty much. See the longer explanation above.